src/store/app-state.ts

```diff
@@ -0,0 +1,17 @@
+import { ref, computed } from 'vue'
+
+const state = ref({
+    step: 1,
```

Comment:
add `firstStep` and `lastStep` variables. `lastStep` should be reactive.
Use these variables in https://github.com/citation-file-format/cffinit/blob/25d068c0a8421c4c1054587fdb61f9fbdc0e62bd/src/store/step.ts#L10

Reply:
done